lib/analyzer/types.ts

```diff
@@ -28,6 +28,7 @@ export enum AnalysisType {
   Apt = "Apt",
   Rpm = "Rpm",
   Binaries = "binaries",
+  Linux = "Linux", // default/unknown/tech-debt
```

Comment:
is it ok you changed the from `linux` to `Linux`?

Reply:
I want to say "no", but the tests pass 🤷‍♂ I'll look into it, good find! thanks

Reply:
in the mean time, restored to ```linux```